examples/align-advanced/CMakeLists.txt

```diff
@@ -2,18 +2,13 @@
 # Copyright(c) 2019-2024 Intel Corporation. All Rights Reserved.
 cmake_minimum_required(VERSION 3.8)
 
-project( rs-align-advanced )
-
-if( BUILD_GRAPHICAL_EXAMPLES )
-
-add_executable( ${PROJECT_NAME} rs-align-advanced.cpp
-    ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp
-    )
-set_property( TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11 )
-target_link_libraries( ${PROJECT_NAME} ${DEPENDENCIES} )
-include_directories( ../../third-party/imgui ../../examples )
-set_target_properties( ${PROJECT_NAME} PROPERTIES FOLDER Examples )
-using_easyloggingpp( ${PROJECT_NAME} SHARED )
-install( TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} )
-
+project(RealsenseExamplesAlignAdvanced)
+
+if(BUILD_GRAPHICAL_EXAMPLES)
+    add_executable(rs-align-advanced rs-align-advanced.cpp ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp  ../../third-party/imgui/realsense_imgui.cpp ../../third-party/imgui/imgui_impl_opengl3.cpp ../../third-party/imgui/imgui_widgets.cpp ../../third-party/imgui/imgui_tables.cpp)
```

Comment:
This looks like a very long list, we will have to do something about it before merging..

Reply:
I added this line in the tools and examples CMakeLists
file(GLOB IMGUI_SOURCES "../third-party/imgui/*.cpp") 
and now all imgui cpp source files are stored in the IMGUI_SOURCES variable and can be used in the CMakelists of each tool and example if needed